scripts/cloudbuild.sh

```diff
@@ -20,7 +20,7 @@ projectRoot=`dirname $0`/..
 
 DOCKER_NAMESPACE='gcr.io/$PROJECT_ID'
 RUNTIME_NAME="openjdk"
-CANDIDATE_NAME="8-`date +%Y-%m-%d_%H_%M`"
+export CANDIDATE_NAME="8-`date +%Y-%m-%d-%H-%M`"
```

Comment:
Why is it `CANDIDATE_NAME` rather than `LABEL`?    Other than that, this is good and builds fine for me.